api/pkg/apps/webapp/individuals.go

```diff
@@ -48,9 +54,23 @@ func (s *Server) Individuals(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	var beneficiaryWithStatuses []IndividualWithStatuses
+	for _, b := range list.Items {
+		rc, err := s.GetRegistrationController(w, req, b)
+		if err != nil {
+			s.Error(w, err)
+			return
+		}
+		beneficiaryWithStatuses = append(beneficiaryWithStatuses, IndividualWithStatuses{
+			Individual:         b,
+			RegistrationStatus: rc.Status(),
+		})
+	}
+
 	if err := s.renderFactory.New(req, w).ExecuteTemplate(w, "individuals", map[string]interface{}{
-		"Individuals": list,
-		"Page":        "list",
+		"Individuals":             list,
+		"IndividualsWithStatuses": beneficiaryWithStatuses,
+		"Page":                    "list",
```

Comment:
Is it necessary to pass in an entirely new pipe?

```
type Individual struct {
    *iam.Individual
    Status *registrationctrl.Status
}
```

Then

```
{{range .Individuals}}
    ...
    {{if .Status}}...{{end}}
{{end}}
```

Reply:
Currently, `Individuals` is an individual list.

The pagination relies on that in the following code segment:
```
{{template "pagination-navigation" .Individuals.Metadata }}
```

I had initially tried to just pass a map of `id` -> `registrationctrl.Status` and then simply call that from the individual row. But I wasn't able to get that working.

Reply:
Ah right right, well in that case I think your solution is fine